src/components/data-management/ProjectDetails.jsx

```diff
@@ -327,7 +327,7 @@ const ProjectDetails = ({ width, height }) => {
 
   const launchAnalysis = (experimentId) => {
     dispatch(runGem2s(experimentId));
-    router.push(analysisPath.replace('[experimentId]', experimentId));
+    window.location.href = analysisPath.replace('[experimentId]', experimentId);
```

Comment:
this causes a hard redirect (i.e. the browser navigates to a new link) instead of a soft redirect. this is needed so we can fetch experiment or backend data from the server-side.

this should also get rid of some of the weird states we have where sometimes redirection to the data processing page causes a 500 error